### PR TITLE
MODDICONV-228 Delete Authority: Create default job profile

### PR DESCRIPTION
**Pull request by ALiemeshka**
## Purpose
The default profile is needed to use delete Authority flow in quickMarc

## Approach
Create sql scripts for Delete profiles

## Learning
[MODDICONV-228](https://issues.folio.org/browse/MODDICONV-228)

### CODE REVIEW

File: mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    "updatedByUserId": "00000000-0000-0000-0000-000000000000"
+  }
+}')
+ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.match_profiles (id, jsonb) values
+('4be5d1d2-1f5a-42ff-a9bd-fc90609d94b6',
+'{
+  "id": "4be5d1d2-1f5a-42ff-a9bd-fc90609d94b6",
+  "name": "Default - Delete MARC Authority",
```
Comment (psmagin) [inline mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql:84]: I think there is some mistake in `name` - no specifying that it's a matching profile. Could you please clarify it with PO?

File: mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            "label": "indicator1",
+            "value": "f"
+          },
+          {
+            "label": "indicator2",
+            "value": "f"
+          },
+          {
+            "label": "recordSubfield",
+            "value": "$s"
```
Comment (psmagin) [inline mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql:109]:
```suggestion
            "value": "s"
```
I think that `$` sign is not needed there.

File: mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
```
Comment (psmagin) [inline mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql:131]:
```suggestion
            "value": "s"
```

File: mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql (hunk 1 above)
Comment (ALiemeshka) [reply mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql:84]: This has already been discussed, "name" is correct

**Review by natali-zaitseva — APPROVED** [review]

**Review by viacheslavkol — APPROVED** [review]

**Review by psmagin — APPROVED** [review]

**Pull request merged**